### PR TITLE
serverspec: add serverspec files

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: td-agent/apt/install-test.sh [hunk 1]
```diff
@@ -59,3 +59,14 @@ piuparts --distribution=${code_name} \
 	 --mirror="http://packages.treasuredata.com/4/${distribution}/${code_name}/ ${code_name} contrib" \
 	 --skip-logrotatefiles-test \
 	 /tmp/*_${architecture}.deb
+
+/usr/sbin/td-agent-gem install serverspec
+wget -qO - https://packages.confluent.io/deb/5.5/archive.key | apt-key add -
+echo "deb [arch=${architecture}] https://packages.confluent.io/deb/5.5 stable main" > /etc/apt/sources.list.d/confluent.list
+apt update && apt install confluent-community-2.12
```
Comment (cosmo0920) [inline td-agent/apt/install-test.sh:66]:
```
apt install confluent-community-2.12
```

Should we use `apt install -y`?

**Comment by cosmo0920** [comment]
Hmm..., I couldn't get why creating kafka topic should be failed in installation testing script...

File: td-agent/apt/install-test.sh [hunk 2]
```diff
@@ -59,3 +66,29 @@ piuparts --distribution=${code_name} \
...
+while true ; do
+    sleep 1
+    status=$(/usr/bin/zookeeper-shell localhost:2181 ls /brokers/ids | sed -n 6p)
+    if [ "$status" = "[0]" ]; then
+	break
+    fi
+done
+/usr/bin/kafka-topics --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic test
+/usr/sbin/td-agent -c /fluentd/serverspec/test.conf &
+cd /fluentd/td-agent && /opt/td-agent/bin/rake serverspec:linux
```
Comment (cosmo0920) [inline td-agent/apt/install-test.sh:94]: Really?
`serverspec:linux` is defined in /fluent/Rakefile.

File: td-agent/yum/install-test.sh [hunk 3]
```diff
@@ -75,3 +82,47 @@ EOF
...
+    while true ; do
+	sleep 1
+	status=$(/usr/bin/zookeeper-shell localhost:2181 ls /brokers/ids | sed -n 6p)
+	if [ "$status" = "[0]" ]; then
+	    break
+	fi
+    done
+    /usr/bin/kafka-topics --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic test
+    /usr/sbin/td-agent -c /fluentd/serverspec/test.conf &
+    cd /fluentd/td-agent && /opt/td-agent/bin/rake serverspec:linux
```
Comment (cosmo0920) [inline td-agent/yum/install-test.sh:127]: ditto.

**Comment by cosmo0920** [comment]
It seems that inside testing docker image does not handle `GITHUB_WORKFLOW` environment variable:
https://github.com/fluent-plugins-nursery/td-agent-builder/runs/1005931111?check_suite_focus=true#step:6:3439

**Comment by kenhys** [comment]
Hmm, It is strange.

```
An error occurred while loading ./serverspec/linux/td-agent.rb.
Failure/Error: raise GemfileNotFound, "Could not locate Gemfile or .bundle/ directory" unless bundle_dir
Bundler::GemfileNotFound:
  Could not locate Gemfile or .bundle/ directory
  
# ./serverspec/linux/td-agent.rb:22:in `new'
# ./serverspec/linux/td-agent.rb:22:in `block in <top (required)>'
# ./serverspec/linux/td-agent.rb:19:in `<top (required)>'
# ------------------
# --- Caused by: ---
# Bundler::GemfileNotFound:
#   Could not locate Gemfile
#   ./serverspec/linux/td-agent.rb:22:in `new'

Finished in 0.00004 seconds (files took 0.93777 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

**Comment by kenhys** [comment]
It may need to check  ` parser = Bundler::LockfileParser.new(Bundler.read_file(lock_path))` lock_path.

**Review by cosmo0920 — APPROVED** [review]
LGTM!

**Comment by cosmo0920** [comment]
I'd merged. Thank you! :muscle:

**Pull request merged**